packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -573,12 +574,14 @@ class _PopupMenu<T> extends StatelessWidget {
     required this.route,
     required this.semanticLabel,
     this.constraints,
+    this.itemKeys,
     required this.clipBehavior,
   });
 
   final _PopupMenuRoute<T> route;
   final String? semanticLabel;
   final BoxConstraints? constraints;
+  final List<GlobalKey>? itemKeys;
```

Comment:
Looks like this can be non-nullable now too?

Reply:
Yes, just updated. Thanks!!